.gitmodules

```diff
@@ -34,3 +34,6 @@
 [submodule "modules/DroneCAN/libcanard"]
 	path = modules/DroneCAN/libcanard
 	url = https://github.com/DroneCAN/libcanard
+[submodule "modules/trusted_flight"]
+	path = modules/trusted_flight
+	url = https://github.com/GlitchedPolygons/l8w8jwt.git
```

Comment:
needs to be forked into ArduPilot github organisation

Reply:
If it gets approved I will help creating a fork in ardupilot